app/services/push_notification_service.rb

```diff
@@ -18,16 +18,34 @@ def send_push_notification
                 "You have #{notifications_count} notifications about posts and #{messages_count} new messages"
               end
 
-    options = {notification: {title: 'Daily Notification', body: message}}
+    options = {title: 'Daily Notification', body: message}
     send(registration_ids, options)
   end
 
   private
 
+
+
   def send(registration_ids, options)
-    fcm = FCM.new(ENV['FCM_API_KEY'])
-    registration_ids.each_slice(20) do |registration_ids_slice|
-      response = fcm.send_notification(registration_ids_slice, options)
+    fcm = FCM.new(ENV['FCM_API_KEY'], Rails.root.join('config', 'credentials', 'firebase_service.json'), 'needpedia-phone-app')
```

Comment:
can you share the 'firebase_service.json' file or provide instructions how to put this on staging and production servers? I see this is part of gitignore so I don't know how to get/create such file, what should be it's content?